test/source/mock/fes/shared-tenant-fes-endpoints.ts

```diff
@@ -0,0 +1,128 @@
+/* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
+
+import { expect } from 'chai';
+import { IncomingMessage } from 'http';
+import { HandlersDefinition } from '../all-apis-mock';
+import { HttpClientErr, Status } from '../lib/api';
+import { MockJwt } from '../lib/oauth';
+import { mockBackendData } from '../backend/backend-endpoints';
+
+const standardFesUrl = 'fes.standardsubdomainfes.localhost:8001';
```

Comment:
This is not relevant in shared-tenant-fes context. That's url of a customer deployed fes.